app/assets/javascripts/host_edit.js

```diff
@@ -2,6 +2,47 @@ $(document).on('ContentLoad', function(){onHostEditLoad()});
 $(document).on('AddedClass', function(event, link){load_puppet_class_parameters(link)});
 $(document).on('click', '#params-tab', function() { resizeTextareas($('#params')); });
 
+LookupValueController = function(item) {
```

Comment:
Extra get request?  Aren't these JS files all combined down into one minified file?  The extra GET would only be present in development mode.  Splitting this out into its own file would make it easier to find in the future.